scripts/scil_visualize_seeds.py

```diff
@@ -2,8 +2,10 @@
 # -*- coding: utf-8 -*-
 
 """
-Visualize seeds where streamlines originate from
-in a tractogram
+Visualize seeds where streamlines originate from a tractogram.
```

Comment:
That sound so weird... 

Visualize seeds used to generate the tractogram or bundle. i.e. when tractography was run, each streamline produced by the tracking algorithm saved its seeding point (its origin).